packages/eslint-plugin-antfu/src/index.ts

```diff
@@ -11,5 +14,8 @@ export default {
     'prefer-inline-type-import': preferInlineTypeImport,
     'generic-spacing': genericSpacing,
     'top-level-function': topLevelFunction,
+    'no-exports': noExports,
+    'no-module-exports': noModuleExports,
+    'no-ts-export': noTsExport,
```

Comment:
```suggestion
    'no-cjs-exports': noExports,
    'no-ts-export-equal': noTsExport,
```

I feel the first two can be merged into one.